content/applications/sales/sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -2,51 +2,77 @@
+
+Then, click the :guilabel:`Save` button in the top-left corner.
 
 .. note::
-   On quotation templates, the :guilabel:`Online Signature` feature is located under the
-   :guilabel:`Confirmation` tab.
+   When making a quotation template, the online signature feature is the :guilabel:`Signature`
+   option, located in the :guilabel:`Online confirmation` field of the quotation template form.
 
-   .. image:: get_signature_to_validate/signature-confirmation-tab.png
+   .. image:: get_signature_to_validate/signature-feature-quotation-template.png
       :align: center
-      :alt: How to enable online signature on a quotation template.
+      :alt: The Online confirmation signature option found on every quotation template in Odoo.
 
-   On standard quotations, the :guilabel:`Online Signature` feature is located under the
-   :guilabel:`Other Info` tab.
+   On standard quotations, the online signature feature is the :guilabel:`Signature` option, located
+   under the :guilabel:`Other Info` tab of the quotation form.
 
    .. image:: get_signature_to_validate/signature-other-info-tab.png
       :align: center
-      :alt: How to enable online signature on standard quotations.
+      :alt: The online signature feature option in the Other Info tab of a quotation form in Odoo.
 
 Order confirmations with online signatures
 ==========================================
 
-When quotations are sent to clients, there's an option for them to :guilabel:`Sign & Pay` directly
-on the quotation in the online customer portal. When clicked, clients have the option to draw a
-signature, fill in the field with an automated signature, or load a file from their computer.
+When clients access quotations online through their customer portal, there's a :guilabel:`Sign &
+Pay` button directly on the quotation.
+
+.. image:: get_signature_to_validate/sign-and-pay-button.png
+   :align: center
+   :alt: The Sign and Pay button present on online quotations in Odoo Sales.
+
+When clicked, a :guilabel:`Validate Order` pop-up window appears. In this pop-up window, the
+:guilabel:`Full Name` field is auto-populated, based on the contact information in the database.
 
-.. image:: get_signature_to_validate/signature-validate-order.png
+.. image:: get_signature_to_validate/validate-order-popup.png
    :align: center
-   :alt: How to confirm an order with a signature on Odoo Sales.
+   :alt: The Validate Order pop-up window for online signatures in Odoo Sales.
+
+Then, customers have the option to enter an online signature with any of the following options:
+:guilabel:`Auto`, :guilabel:`Draw`, or :guilabel:`Load`.
+
+:guilabel:`Auto` lets Odoo automatically generate an online signature based on the information in
+the :guilabel:`Full Name` field. :guilabel:`Draw` lets the customer use the cursor to create a
+custom signature directly on the pop-up window. And :guilabel:`Load` lets the customer upload a
+previously-created signature file from their computer.
```

Comment:
Do we have restrictions/guidelines for uploading a signature file? (I honestly do not know if there are restrictions and I tried uploading a dummy png file just to test in the portal and it seemed to work, so I do not know if there are any requirements or not).

Reply:
Great question! I don't believe we have any restrictions, nor any specific guidelines, for uploading a signature file. However, if any user leaves any comment about a potential issue they may have, I can always adapt this doc to adhere to their problem. But, given that we don't (at the moment) have anything like that to worry about, I think I can leave this as-is. But thanks for bringing this to my attention! 👍